File: check_regex.py
import re

with open('c:\\workspace\\itstory\\os.html', 'r', encoding='utf-8') as f:
    content = f.read()

print("timeline-group__summary:", len(re.findall(r'<p class="timeline-group__summary">.*?</p>', content, flags=re.DOTALL)))
print("event-detail__summary:", len(re.findall(r'<p class="event-detail__summary">.*?</p>', content, flags=re.DOTALL)))
print("event-detail__story:", len(re.findall(r'<div class="event-detail__story">.*?</div>', content, flags=re.DOTALL)))
print("event-detail__notes:", len(re.findall(r'<div class="event-detail__notes">.*?</div>', content, flags=re.DOTALL)))

File: scripts/check_regex.js
const fs = require('fs');
const content = fs.readFileSync('c:\\workspace\\itstory\\os.html', 'utf-8');

console.log('timeline-group__summary:', (content.match(/<p class="timeline-group__summary">[\s\S]*?<\/p>/g) || []).length);
console.log('event-detail__summary:', (content.match(/<p class="event-detail__summary">[\s\S]*?<\/p>/g) || []).length);
console.log('event-detail__story:', (content.match(/<div class="event-detail__story">[\s\S]*?<\/div>/g) || []).length);
console.log('event-detail__notes:', (content.match(/<div class="event-detail__notes">[\s\S]*?<\/div>/g) || []).length);
